library/spdm_responder_lib/libspdm_rsp_capabilities.c

```diff
@@ -154,23 +145,20 @@ libspdm_return_t libspdm_get_response_capabilities(void *context,
                                                response_size, response);
     }
     if (spdm_request->header.spdm_version >= SPDM_MESSAGE_VERSION_12) {
-        if (request_size != sizeof(spdm_get_capabilities_request_t)) {
+        if (request_size > sizeof(spdm_get_capabilities_request_t)) {
```

Comment:
why bigger is problem?

I think the general flow is:
```
    if (size < sizeof(xxx_request_t)) {
        return libspdm_generate_error_response ();
    } else {
        size = sizeof(xxx_request_t);
    }
```

Reply:
Thanks. Got my direction wrong.